Jenkinsfile

```diff
@@ -0,0 +1,54 @@
+pipeline {
+    agent none
+    environment {
+        projectName = 'DDF'
+    }
+    stages {
+        stage('Setup') {
+            steps{
+                notifySlack('good', "STARTED: ${JOB_NAME} ${BUILD_NUMBER} ${BUILD_URL}")
+            }
+        }
+        stage('Parallel Build') {
+            // TODO refactor this stage from scripted syntax to declarative syntax to match the rest of the stages - https://issues.jenkins-ci.org/browse/JENKINS-41334
+            steps{
+                parallel(
+                    linux: {
+                        node('linux-large') {
```

Comment:
When declaring the agent at the steps level it is that format, but inside the parallel block it uses the scripted `node(` syntax. This should change when they get the `parallelSteps` command change merge in.